services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/ControllerTest.kt

```diff
@@ -24,6 +24,13 @@ interface ControllerTest {
             .andExpect(content().contentType(resultType))
     }
 
+    /** Send a GET request to the given URL without expecting content of certain type. */
+    fun getRaw(
```

Comment:
`get` could also be modified to accept `null`.

```kotlin
    fun get(
        url: String,
        resultType: MediaType? = MediaType.APPLICATION_JSON,
    ): ResultActions {
        val actions =
            mockMvc.perform(MockMvcRequestBuilders.get(url).accept(MediaType.APPLICATION_JSON))

        return if (resultType != null) {
            actions.andExpect(content().contentType(resultType))
        } else {
            actions.andExpect(header().doesNotExist("content-type"))
        }
    }
```

Reply:
Yes, this is better than my choice. Will change.